frameworks/helloworld/tests/test_soak.py

```diff
@@ -65,27 +54,52 @@ def test_soak_secrets_update():
     # make sure content is changed
     assert secret_content_alternative == task_exec(world_tasks[0], "bash -c 'echo $WORLD_SECRET1_ENV'")
     assert secret_content_alternative == task_exec(world_tasks[0], "cat WORLD_SECRET2_FILE")
-    assert secret_content_alternative == task_exec(world_tasks[0], "cat {}/secret3".format(PACKAGE_NAME))
+    assert secret_content_alternative == task_exec(world_tasks[0], "cat secrets/secret3")
 
     # make sure content is changed
     assert secret_content_alternative == task_exec(hello_tasks[0], "bash -c 'echo $HELLO_SECRET1_ENV'")
     assert secret_content_alternative == task_exec(hello_tasks[0], "cat HELLO_SECRET1_FILE")
     assert secret_content_alternative == task_exec(hello_tasks[0], "cat HELLO_SECRET2_FILE")
 
-    # revert back
+    # revert back to some other value
     cmd.run_cli("security secrets update --value=SECRET1 secrets/secret1")
     cmd.run_cli("security secrets update --value=SECRET2 secrets/secret2")
     cmd.run_cli("security secrets update --value=SECRET3 secrets/secret3")
+    test_soak_secrets_restart_hello0()
+
+
+@pytest.mark.soak_secrets_alive
+@dcos_1_10
```

Comment:
already pytest.mark

dcos_1_10 = pytest.mark.skipif('dcos_version_less_than("1.10")')
https://github.com/dcos/shakedown/blob/master/shakedown/dcos/cluster.py

May be using the following is easier:
@pytest.mark.skipif('shakedown.dcos_version_less_than("1.9")')

But documented way of doing is https://github.com/dcos/shakedown/blob/master/API.md#dcos_1_10